test/language/expressions/class/elements/static-as-valid-instance-field.js

```diff
@@ -0,0 +1,36 @@
+// This file was procedurally generated from the following sources:
+// - src/class-elements/static-as-valid-instance-field.case
+// - src/class-elements/default/cls-expr.template
+/*---
+description: static is a valid name of an instance field (field definitions in a class expression)
+esid: prod-FieldDefinition
+features: [class-fields-public, class]
+flags: [generated]
+includes: [propertyHelper.js]
+info: |
+    ClassElement:
+      ...
+      FieldDefinition ;
+
+    FieldDefinition:
+      ClassElementName Initializer_opt
+
+    ClassElementName:
+      PropertyName
+
+---*/
+
+
+var C = class {
+  static = "foo";
```

Comment:
should this also verify a `static static = ` field?

Reply:
Nice catch. I think we should do this in a separate file/PR, since static fields uses a different flag than `class-fields-public`